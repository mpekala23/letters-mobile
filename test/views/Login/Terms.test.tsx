import * as React from 'react';
import { TermsScreen } from '@views';
import { render, toJSON } from '@testing-library/react-native';

describe('Terms of Service screen', () => {
  it('should match snapshot', () => {
    const { container } = render(<TermsScreen />);
    const tree = toJSON(container);
    expect(tree).toMatchSnapshot();
  });
});
