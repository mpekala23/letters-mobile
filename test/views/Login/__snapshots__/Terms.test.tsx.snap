// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terms of Service screen should match snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "flex": 1,
        "justifyContent": "center",
        "padding": 16,
      }
    }
  >
    <Text>
      Introduction. To assist you in using our Services, and to ensure a clear understanding of the relationship from your use of the Services, we have created these Terms of Services (the “Terms”). The Terms constitute a binding agreement between you and Ameelio. The Terms govern your use of the Services and any other services offered as part of the Ameelio platform. The Terms also governs any information, text, graphics, photos, recordings, or other materials uploaded, downloaded, or appearing on the Services (collectively referred to as “Content”).
    </Text>
  </View>
</View>
`;
